src/app/styles.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import image from '../../public/HomePage.webp';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+
+export const TitleSection = styled.div`
+  width: 100%;
+  height: 100vh; // Use 100vh to make the div fill the entire viewport height
+  background: linear-gradient(
+      222deg,
+      rgba(40, 120, 194, 0.5) 14.2%,
+      rgba(0, 50, 96, 0.5) 76.27%
+    ),
+    url(${image.src}) no-repeat center / cover;
```

Comment:
Instead of a background image, I would strongly recommend simply putting the Image tag in the HTML and overlaying the text on top of it using `position: absolute` and setting the `z-index` to be some positive number. Background images do not take advantage of Next.js' image optimization and the lack of a background image could look really nasty on devices with slower internet.